tests/integration_tests/performance/test_network_tcp_throughput.py

```diff
@@ -13,81 +13,93 @@
     DEFAULT_HOST_IP
 from framework.matrix import TestMatrix, TestContext
 from framework.builder import MicrovmBuilder
-from framework.statistics import core, consumer, producer, criteria,  types, \
-    function
+from framework.statistics import core, consumer, producer, criteria, types
 from framework.utils import CpuMap, CmdBuilder, run_cmd, eager_map, \
     get_cpu_percent
 from framework.utils_cpuid import get_cpu_model_name
 import host_tools.network as net_tools
-from integration_tests.performance.network_tcp_throughput_test_config import \
-    THROUGHPUT, DURATION, RETRANSMITS, CPU_UTILIZATION_HOST, \
-    CPU_UTILIZATION_GUEST, CONFIG, CPU_UTILIZATION_VMM_TAG, \
-    CPU_UTILIZATION_VCPUS_TOTAL_TAG, THROUGHPUT_TOTAL, RETRANSMITS_TOTAL, \
-    DURATION_TOTAL, IPERF3, BASE_PORT, DEBUG, \
-    IPERF3_CPU_UTILIZATION_PERCENT_OUT_TAG, IPERF3_END_RESULTS_TAG, \
-    DEBUG_CPU_UTILIZATION_VMM_SAMPLES_TAG, DELTA_PERCENTAGE_TAG, TARGET_TAG
+from integration_tests.performance.configs.network_tcp_throughput_test_config \
```

Comment:
Can we just use
```python
import integration_tests.performance.configs.network_tcp_throughput_test_config as test_cfg
```

And further down just use `test_cfg.THROUGHPUT` and so on.

WDYT?